src/mixins/filing-template-mixin.ts

```diff
@@ -452,14 +447,11 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
     this.setEntityType(continuationIn.nameRequest.legalType)
 
     // restore existing business information
-    if (
-      continuationIn.business &&
-      continuationIn.foreignJurisdiction
-    ) {
+    if (continuationIn.foreignJurisdiction) {
       this.setExistingBusinessInfo({
         affidavitFileKey: continuationIn.foreignJurisdiction.affidavitFileKey,
-        businessIdentifier: continuationIn.business.identifier,
-        businessLegalName: continuationIn.business.legalName,
+        businessIdentifier: continuationIn.business?.identifier, // FUTURE: need fallback to null?
+        businessLegalName: continuationIn.business?.legalName, // FUTURE: need fallback to null?
```

Comment:
Ketaki, I had second thoughts about this... If there was no business object, should these fields be null or undefined?

Probably need to test without an expro (ie, your ticket) and see what makes the most sense. For example, does everything still work if the user clicks Undo on the manual component and then searches for an expro?

Reply:
Okay I will keep a note of this and test it